pkg/spec/createconfig.go

```diff
@@ -510,6 +510,7 @@ func (c *CreateConfig) GetContainerCreateOptions(runtime *libpod.Runtime) ([]lib
 	options = append(options, libpod.WithShmSize(c.Resources.ShmSize))
 	options = append(options, libpod.WithGroups(c.GroupAdd))
 	options = append(options, libpod.WithIDMappings(*c.IDMappings))
+	options = append(options, libpod.WithContainerOCIRuntime(c.Runtime.GetConfig().OCIRuntime))
```

Comment:
Instead of this, can we set it to the default OCI runtime when we create the container in runtime_ctr.go? I'm fine with leaving WithContainerOCIRuntime, but we should set the field to something reasonable by default